<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Serial Query Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourcePathDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&quot;Source File&quot;</string>
    </property>
    <property name="toolTipDuration">
     <number>10</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="DestinationPathDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Destination File (will be recreated)</string>
    </property>
    <property name="toolTipDuration">
     <number>10</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="Button1">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button2">
       <property name="text">
        <string>Show example</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button3">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="List1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
